.github/workflows/qa.yml

```diff
@@ -9,6 +9,6 @@ defaults:
 
 jobs:
     call-qa:
-        uses: symfony/recipes/.github/workflows/callable-qa.yml@main
+        uses: symfony/recipes/.github/workflows/callable-qa.yml@5b2dcec
```

Comment:
Is it needed once merged?

Reply:
No, the test endpoint wasn't updated since we use the git-based recipe update. I was fighting the CI to update it, but can't get it working (maybe requires a separate PR).
I'll manually set-up a test endpoint to test the upgrade and remove these things from the PR.

Reply:
@wouterj `pull_request_target` uses the CI of the main branch so changes of the CI files do not effect the CI run itself. Its kind of security feature but little bit anoying if you need to change the CI files: https://stackoverflow.com/questions/74957218/what-is-the-difference-between-pull-request-and-pull-request-target-event-in-git